src/neptune/version.py

```diff
@@ -0,0 +1,33 @@
+#
+# Copyright (c) 2022, Neptune Labs Sp. z o.o.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import sys
+
+if sys.version_info >= (3, 8):
+    from importlib.metadata import (
+        PackageNotFoundError,
+        version,
+    )
+else:
+    from importlib_metadata import (
+        PackageNotFoundError,
+        version,
+    )
+
+try:
+    __version__ = version("neptune-client")
+except PackageNotFoundError:
```

Comment:
Is possible catch `PackageNotFoundError` when python has the `src/neptune/version.py` in path?

Reply:
`importlib.metdata.version("something")` works similarly to `pip list | grep something` instead of checking the current namespace.
For instance `typing` is a proper builtin package but:
```python
>>> import typing
>>> importlib.metadata.version("typing")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/neptune/.conda/envs/neptune-client/lib/python3.10/importlib/metadata/__init__.py", line 984, in version
    return distribution(distribution_name).version
  File "/Users/neptune/.conda/envs/neptune-client/lib/python3.10/importlib/metadata/__init__.py", line 957, in distribution
    return Distribution.from_name(distribution_name)
  File "/Users/neptune/.conda/envs/neptune-client/lib/python3.10/importlib/metadata/__init__.py", line 548, in from_name
    raise PackageNotFoundError(name)
importlib.metadata.PackageNotFoundError: No package metadata was found for typing
```
so I think that nothing will impact it except `pip install neptune-client` or `pip install -e .` or `pip install .` (including direct call to `python setup.py install`)